Tests/Grid/Mapping/Metadata/ManagerTest.php

```diff
@@ -0,0 +1,161 @@
+use APY\DataGridBundle\Grid\Mapping\Driver\DriverInterface;
+
+class ManagerTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->manager = new Manager();
+    }
+
+    public function testAddDriver()
+    {
+        $driverInterfaceMock = $this->createMock(DriverInterface::class);
+        $priority = 1;
+
+        $driverHeap = new DriverHeap();
+        $driverHeap->insert($driverInterfaceMock, $priority);
+
+        $this->manager->addDriver($driverInterfaceMock, $priority);
+
+        $this->assertAttributeEquals($driverHeap, 'drivers', $this->manager);
+    }
+
+    public function testGetDrivers()
+    {
+        $driverInterfaceMock = $this->createMock(DriverInterface::class);
+        
+        $priority = 1;
+        $driverHeap = new DriverHeap();
+        $driverHeap->insert($driverInterfaceMock, $priority);
+
+        $this->manager->addDriver($driverInterfaceMock, $priority);
+        $drivers = $this->manager->getDrivers();
+        
+        $this->assertEquals($driverHeap, $drivers);
+    }
+
+    public function testGetDriversReturnDifferentClone()
+    {
+        $driverFirstTime = $this->manager->getDrivers();
+        $driverSecondTime = $this->manager->getDrivers();
+
+        $this->assertNotSame($driverFirstTime, $driverSecondTime);
+    }
+
+    public function testWithoutDriverProvideEmptyMetadata()
+    {
+        $cols = [];
+        $mappings = [];
+        $groupBy = [];
+
+        $metadataExpected = new Metadata();
+        $metadataExpected->setFields($cols);
+        $metadataExpected->setFieldsMappings($mappings);
+        $metadataExpected->setGroupBy($groupBy);
+
+        $metadata = $this->manager->getMetadata('foo', 'bar');
+
+        $this->assertEquals($metadataExpected, $metadata);
+    }
+
+    public function testAddDriverProvideMetadata()
+    {
+        $classColumnExpected = ['0' => 'bar'];
+        $groupByExpected = ['foo' => 'bar'];
+        $mappingExpected = [
+            'bar' => [
+                'foo' => 'foo2'
+            ]
+        ];
+
+        $driverInterfaceMock = $this->createMock(DriverInterface::class);
+        $driverInterfaceMock->method('getClassColumns')
+                            ->willReturn($classColumnExpected);
+
+        $driverInterfaceMock->method('getFieldsMetadata')
+                            ->willReturn($mappingExpected);
+
+        $driverInterfaceMock->method('getGroupBy')
+                            ->willReturn($groupByExpected);
+
+        $this->manager->addDriver($driverInterfaceMock, 1);
+
+        $metadataExpected = new Metadata();
+        $metadataExpected->setFields($classColumnExpected);
+        $metadataExpected->setFieldsMappings($mappingExpected);
+        $metadataExpected->setGroupBy($groupByExpected);
+
+        $this->assertEquals(
+            $metadataExpected, 
+            $this->manager->getMetadata('foo')
+        );
+    }
+
+    public function testAddTwoDriversProvideMetadata()
```

Comment:
This test can be collapsed with previous one (that can be removed). As stated above, don't use getter but write a test case (maybe edit the previous one) to point out you're testing the `getter` and not the `adder`